src/vs/base/browser/ui/menu/menu.ts

```diff
@@ -216,6 +216,15 @@ export class Menu extends ActionBar {
 			return true;
 		});
 
+		actions = actions.filter(a => {
```

Comment:
all changes to `vs` source files need to have a `// {{SQL CARBON EDIT}}` tag so we can track patches to vscode source.  I don't understand why this change is needed though.  Could you verify this isn't a merge issue and revert if not needed or add a comment if actually needed?

Reply:
@kburtram-I will take a look at it